crates/nu-cli/src/cli.rs

```diff
@@ -641,10 +657,14 @@ pub fn set_rustyline_configuration() -> (Editor<Helper>, IndexMap<String, Value>
                             _ => rustyline::config::EditMode::Emacs,
                         };
                         rl.set_edit_mode(edit_mode);
+                        if edit_mode == rustyline::config::EditMode::Emacs {
+                            rl.set_keyseq_timeout(-1);
+                        }
```

Comment:
This one looks new. Necessary?

Reply:
Yes, it's new and necessary. I found it while looking through the rustyline code. Apparently, in emacs mode the timeout should be `-1` as I have it coded here. I also noticed that we don't appear to have a `value.as_i32()` so having a `-1` in the config file could cause a problem.

Reply:
ok, i take it back. it's not completely necessary but a good reminder of what's going on in rustyline. emacs expects key chords which is why the timeout is turned off with -1.

Reply:
just looked at rustyline again. i'm going to remove it and put a comment in it's place. good catch.